src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher5.php

```diff
@@ -28,34 +28,23 @@ public function match($rawPathinfo)
             $canonicalMethod = 'GET';
         }
 
-        if (0 === strpos($pathinfo, '/a')) {
-            // a_first
-            if ('/a/11' === $pathinfo) {
+        switch ($trimmedPathinfo) {
+            case '/a/11':
+                // a_first
                 return array('_route' => 'a_first');
-            }
-
-            // a_second
-            if ('/a/22' === $pathinfo) {
+                break;
```

Comment:
This `break` statement seems useless.

Reply:
Thanks @phansys, you missed some :)
This is generated code, removing them would add a lot of complexity for no practical benefit.

Reply:
I wonder if unreachable code has a performance impact, as the opcodes are still generated (but never used).

Reply:
that's opcache's job, removing dead code